### ISSUE TITLE
[bun pm] Display untrusted reason

### ISSUE THREAD

**Issue by sequencerr** [oven-sh/bun#9363]
### What is the problem this feature would solve?

```$ bun install tesseract.js
bun add v1.0.30-canary.38 (cd320835)

 installed tesseract.js@5.0.5

 14 packages installed [5.27s]

 Skipped ~1 script. Run `bun pm trusted` for details.
$ bun pm trusted
Trusted dependencies (0):

Untrusted dependencies (1):
 - tesseract.js
$
```
Related: https://github.com/oven-sh/bun/pull/7857

User may know that there is a reason to not trusting dependency. May know it's not just forgotten and missing in [src/install/default-trusted-dependencies.txt](https://github.com/oven-sh/bun/blob/main/src/install/default-trusted-dependencies.txt). And thus it's maybe no necessity to manually `trust` it nor raising an issue.

### What is the feature you are proposing to solve the problem?

Display reason right after `bun install`'s `Skipped ~1 script`. Or in `bun pm trusted`

**Comment by Jarred-Sumner** [comment]
yeah we are changing the output of this command, @dylan-conway is working on it

**Comment by dylan-conway** [comment]
closed with #9371
